compiler/test/suites/memory_base.re

```diff
@@ -0,0 +1,18 @@
+open Grain_tests.TestFramework;
+open Grain_tests.Runner;
+
+describe("memory_base", ({test}) => {
+  let assertFileRun = makeFileRunner(test);
+  // Non-snapshots for now, since we want to track functional issues.
+  // Maybe these can be migrated to snapshots once we're confident in
+  // everything being stable
```

Comment:
```suggestion
  // Non-snapshots since we want to track functional issues.
```
Hopefully we're confident they're stable now 😅 but yeah I think it makes sense for them to always be actually run.

Reply:
😡 😅